core/ir/aarch64/instr_create.h

```diff
@@ -2544,5 +2544,24 @@ enum {
 #define INSTR_CREATE_umull2_vector(dc, Rd, Rm, Rn, width) \
     instr_create_1dst_3src(dc, OP_umull2, Rd, Rm, Rn, width)
 
+/**
+ * Creates a FMOV immediate to vector floating point move instruction.
```

Comment:
nit: s/a/an/ (unless you pronounce it "fuh-move"?  I guess I see it as "eff-move")

Reply:
Done.